infra/chart/values.yaml

```diff
@@ -102,8 +102,13 @@ dbMigrations:
     tag: v4.7.1
 
 swaggerUi:
-  # -- Docker tag for Swagger UI https://hub.docker.com/r/swaggerapi/swagger-ui
   image:
+    # -- Docker registry for Turing Swagger UI image. User is required to override
+    # the registry for now as there is no publicly available Turing Swagger UI image
```

Comment:
Can you include a GH workflow (or a step in existing turing-api workflow) and publish image to ghcr.io?

Reply:
Turing also?

Reply:
I've done just for Swagger only.